packages/jsActions/mobile-resources-native/package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/mendix/widgets-resources.git"
   },
   "marketplace": {
-    "minimumMXVersion": "9.0.5.17836",
+    "minimumMXVersion": "9.6.0.27784",
```

Comment:
I don't think we need to bump the minimum version. The module is still backwards compatible, right? Older versions will just ignore the images. Not sure what the impact is of the categories though..

Reply:
Older versions of Studio Pro will not recognize new XML tags like  `<studioProCategory>`, so the widget will not be backwards compatible.

Reply:
If it breaks on import, then I understand the need for the bump